test/acceptance/coffee/helpers/TagsClient.coffee

```diff
@@ -0,0 +1,60 @@
+settings = require("settings-sharelatex")
+request = require("request").defaults({
+	jar: false,
+	baseUrl: "http://localhost:#{settings.internal.tags.port}",
+})
+
+module.exports =
+	getUserTags: (user_id, callback = (error, res, body) ->) ->
```

Comment:
There is a different default-initialiser config here than elsewhere. - This one has `error, res, body` where the rest just have `error`. (I realise that this is a getter where the others are setters, but when calling this method I would probably still want the `statusCode` from `res`.)

Personally I don't think these initialisers are necessary. When we were using 100% coffeescript they served as documentation on what callback parameters to expect, but in practise they don't add much value and end up generating messy artifacts when we get around to decaffeinating the project.

I'd be inclined to remove them here. It's a nitpick though. :-)

Reply:
I'm fine with removing them!